packages/Acme/Domain/VTuber.php

```diff
@@ -0,0 +1,67 @@
+<?php
+declare(strict_types=1);
+
+namespace Acme\Domain;
+
+class VTuber
+{
+
+    private const TWITTER_MINIMUM_AGE = 13;
+    /**
+     * @var string
+     */
+    private $name;
+    /**
+     * @var bool
+     */
+    private $isBanned;
+    /**
+     * @var int
+     */
+    private $age;
+
+    public function __construct(string $name, int $age, bool $isBanned)
+    {
+        $this->name = $name;
+        $this->age = $age;
+        if ($age < self::TWITTER_MINIMUM_AGE && !$isBanned) {
+            throw new \LogicException('Twitterがなんかおかしい');
+        }
+    }
+
+    public function thisFunctionShouldBeWatchedByDog(): string
+    {
+        return $this->age;
+    }
+
+    public function thisFunctionShouldBeWatchedByDog2(): string
+    {
+        return $this->age;
```

Comment:
**[phpstan]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
Method Acme\Domain\VTuber::thisFunctionShouldBeWatchedByDog2() should return string but returns int.